recipe/meta.yaml

```
@@ -10,35 +8,49 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 77891101c94b75a8ca305453b879b318ab6001b370df02be2c0b6d1bb322db10
 
 build:
-  noarch: python
+  skip: True  # [py<37]
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
-    - pytest >=3.0
+    - setuptools
+    - wheel
+    - setuptools_scm
   run:
     - python
-    - pytest >=3.0
+    - pytest >=4.6
 
 test:
+  source_files:
+    - tests/
   imports:
     - pytest_astropy_header
     - pytest_astropy_header.display
+  requires:
+    - pip
+    - pytest
+    - numpy
```

```suggestion
  requires:
    - pip
    - pytest
    - numpy
    - astropy >=4.0
```

From https://github.com/astropy/pytest-astropy-header/blob/v0.2.2/setup.cfg#L33

---

We either have to figure out the entry point or leave out `astropy` from tests. They did run without as you can see.

---

Lmk if you'd like to help with the solution

---

Having dug into it a bit, I think upstream put that in by mistake. I see no usage of astropy in any of their tests and the test suite runs with or without it present. Whether or not the entrypoint needs to be created, though, is still a question.

---

Only the console_scripts entry points. Other types of entrypoints are included in the metadata of the package.